DashAI/back/dataloaders/classes/csv_dataloader.py

```diff
@@ -0,0 +1,35 @@
+import io
+import os
+import zipfile
+
+from datasets import load_dataset
+
+from DashAI.back.dataloaders.classes.dataloader import DataLoader
+
+
+class CSVDataLoader(DataLoader):
+    """
+    Data loader for tabular data in CSV files
+    """
+
+    def load_data(self, dataset_path, params, file=None, url=None):
```

Comment:
Is necessary the parameters in the params dictionary, because if you pass as explicit arguments you cannot generalize the parameters for all data loaders in the endpoint. If it were explicit arguments you would have to pass specifically the parameter it needs each dataloader, and with the dictionary you can pass any parameter it needs and this is internally obtained from the dictionary, so each dataloader gets the parameter it needs without having to make extra conditionals for each dataloader in the endpoint. 
```
dataset = dataloader.load_data(
            dataset_path=folder_path,
            params=params.dataloader_params.dict(),
            file=file,
            url=url,
        )
```
This is considering that the 'dataloader' called in this code changes depending of the user input, here is called the load method of the dataloader chosen by the user, so the parameters are also variable. And this was the solution that I found more convenient.